### ISSUE TITLE
multiple GPUs

### ISSUE THREAD

**Issue by diamond0910** [eladrich/pixel2style2pixel#308]
pSp has BatchNorm2 layer. Do multiple GPUs need to change it to SyncBatchNorm？

**Comment by yuval-alaluf** [comment]
Most likely. If you're using multiple GPUs you should use the best practices for multi-gpu training.

**Comment by LightingMc** [comment]
Any updates on this? I tried doing the SyncBatchNorm thing, but mine gets stuck.

**Comment by diamond0910** [comment]
> Any updates on this? I tried doing the SyncBatchNorm thing, but mine gets stuck.

I don't have a try. It seems all the gan inversion methods use single gpu to train for serval days. If you success in multi gpus, please tell me.

**Comment by yuval-alaluf** [comment]
I would refer to this thread here since it seems like someone was able to get it working: https://github.com/eladrich/pixel2style2pixel/issues/157
We are not planning on implementing multi-GPU training since the speed-up you'll get in doing so will be rather minimal. 
If you want to use multi-GPU, my only recommendation is to refer to other threads on how to do so effectively using Pytorch. 
Since this is not an issue with the code base I'm closing this issue.
